docs/pages/enroll-resources/database-access/auto-user-provisioning/mariadb.mdx

```diff
@@ -38,14 +38,14 @@ This database is also used to store custom user attributes and stored procedures
 The RDS MariaDB admin user must use `AWSAuthenticationPlugin` to allow IAM
 authentication:
 ```sql
-CREATE USER 'teleport-admin' IDENTIFIED WITH AWSAuthenticationPlugin AS 'RDS';
-GRANT PROCESS, CREATE USER ON *.* TO 'teleport-admin';
-GRANT SELECT ON mysql.roles_mapping TO 'teleport-admin';
-GRANT UPDATE ON mysql.* TO 'teleport-admin'; -- For SET DEFAULT ROLE FOR
-GRANT SELECT ON *.* TO 'teleport-admin'; -- Required when using best_effort_drop mode for checking if users own resources before dropping them.
+CREATE USER '<Var name="teleport-admin"/>' IDENTIFIED WITH AWSAuthenticationPlugin AS 'RDS';
```

Comment:
this makes it harder to read the sql script and copy-pasting individual lines/section.

i think `teleport-admin` is a good name most of the time. any reason why a user needs to customize to a different name? if it's a naming conflict, we can add prefix support to auto-provisioned user

or, i don't know if there is "muted" version of the var that's non-editable inside code block but we can use the editable version outside